app/dossierPrograms/dossierPrograms.controllers.js

```diff
@@ -206,6 +206,202 @@ dossierProgramsModule.controller("dossiersProgramSectionController", [
+                    update: true,
+                    externalize: true,
+                    delete: true,
+                    write: true,
+                    manage: true,
+                },
+                reportingParams: {
+                    organisationUnit: false,
+                    reportingPeriod: false,
+                    parentOrganisationUnit: false,
+                    grandParentOrganisationUnit: false,
+                },
+
+                axes: [],
+                translations: [],
+                yearlySeries: [],
+                interpretations: [],
+                userGroupAccesses: [
+                    {
+                        access: "rw------",
+                        userGroupUid: "epFY01iJN0Z",
+                        displayName: "ALL USERS",
+                        id: "epFY01iJN0Z",
+                    },
+                ],
+                subscribers: [],
+                userAccesses: [],
+                favorites: [],
+                columns: [
+                    {
+                        dimension: "pe",
+                        items: [
+                            {
+                                id: "THIS_YEAR",
+                            },
+                        ],
+                    },
+                    {
+                        dimension: "BtFXTpKRl6n",
+                        items: [
+                            {
+                                id: "ALL_ITEMS",
+                            },
+                        ],
+                    },
+                ],
+                filters: [
+                    {
+                        dimension: "ou",
+                        items: [
+                            {
+                                id: "USER_ORGUNIT",
+                            },
+                        ],
+                    },
+                ],
+                rows: [
+                    {
+                        dimension: "dx",
+                        items: [
+                            {
+                                id: "DqqSJFWB392",
+                            },
+                            {
+                                id: "qywsusOdy33",
+                            },
+                        ],
+                    },
+                ],
+                series: [],
+                outlierAnalysis: null,
+                cumulative: false,
+            };
+
+            const sharing = {
+                object: {
+                    id: "LEP0WHTGYUe",
+                    name: "name",
+                    publicAccess: "--------",
+                    externalAccess: false,
+                    userGroupAccesses: [{ id: "epFY01iJN0Z", name: "ALL USERS", access: "rw------" }],
+                },
+            };
+
+            let items = [{ id: id }];
+
+            payload.name = name + " - " + id;
+            payload.rows[0].items = rowItems ? items.concat(rowItems.map(id => ({ id: id }))) : items;
```

Comment:
[minor, no need to change] I guess we want a base payload which can be easily copy/pasted, that's why we modify it here. With a functional approach, we would have a `basePayload` and then `const payload = {...basePayload, ...}` with all the required changes.